### PR TITLE
very simple symbolic_trace added

### PR DESCRIPTION
**Pull request by jzhang533**
the current ability can be checked by running usecases/simple_net_trace.py 
lots of TODO still remains

### CODE REVIEW

**Comment by gglin001** [comment]
代码需要 执行一下 pre-commit 做格式化
可以是
```
pre-commit run -a
```

File: src/paddlefx/playground.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from typing import TYPE_CHECKING, Union, Callable, Any, Tuple, List, Optional, Dict
```
Comment (gglin001) [inline src/paddlefx/playground.py:1]: 这个文件有被使用吗? 是不是不应该提上来?

File: src/paddlefx/usecases/simple_net_trace.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import paddle
```
Comment (gglin001) [inline src/paddlefx/usecases/simple_net_trace.py:1]: 将这个 文移动到 `{root_dir}/examples/simple_net_trace.py` 会更合适吗?

File: src/paddlefx/playground.py (hunk 1 above)
Comment (jzhang533) [reply src/paddlefx/playground.py]: deleted

**Comment by jzhang533** [comment]
codes are formatted.

File: src/paddlefx/usecases/simple_net_trace.py (hunk 2 above)
Comment (jzhang533) [reply src/paddlefx/usecases/simple_net_trace.py]: moved

**Review by gglin001 — APPROVED** [review]
LGTM!

**Pull request merged**